apps/www/components/ui/navigation-menu.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react"
+import * as NavigationMenuPrimitive from "@radix-ui/react-navigation-menu"
+import { cva, type VariantProps } from "class-variance-authority"
+import { ChevronDown } from "lucide-react"
+
+import { cn } from "@/lib/utils"
+
+export const NavigationMenuRoot: React.FC<
+  NavigationMenuPrimitive.NavigationMenuProps
+> = ({ className, ...props }) => (
+  <NavigationMenuPrimitive.Root
+    className={cn("relative z-[1] flex flex-1 justify-center", className)}
+    {...props}
+  />
+)
+
+export const NavigationMenuList: React.FC<
+  NavigationMenuPrimitive.NavigationMenuListProps
+> = ({ className, ...props }) => (
+  <NavigationMenuPrimitive.List
+    className={cn(
+      "group flex flex-1 list-none justify-center rounded-lg border border-slate-300 bg-white p-1 dark:border-slate-700 dark:bg-slate-800",
+      className
+    )}
+    {...props}
+  />
+)
+
+export const NavigationMenuItem = NavigationMenuPrimitive.NavigationMenuItem
+
+const navigationMenuTriggerStyle = cva(
```

Comment:
Can we inline the classes in the `NavigationMenuTrigger`? Are we re-using this elsewhere?

Reply:
it's used in the NavigationMenuLink to have the same style by default better to remove it ?